.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: CI Checks
+
+on:
+  push:
+    branches: ["**"]
+  pull_request:
+    branches: [dev]
+  workflow_dispatch:
+
+jobs:
+  complexity:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Can you replace the complexity check with link verifier?

Reply:
Will do